apps/bfd-server/bfd-server-war/src/main/java/gov/cms/bfd/server/war/r4/providers/BeneficiaryTransformerV2.java

```diff
@@ -278,7 +287,7 @@ private static Patient transform(Beneficiary beneficiary, RequestHeaders request
           .setValue(
               new Coding()
                   .setCode(ombCode)
-                  .setSystem("urn:oid:2.16.840.1.113883.6.238")
+                  .setSystem(TransformerConstants.CODING_V3_NULL)
```

Comment:
Was looking at the output.  It appears we do have a race that fits into the "OMB Race Categories" but we are still setting it to `UNK`

```
        {
            "url": "https://bluebutton.cms.gov/resources/variables/race",
            "valueCoding": {
                "system": "https://bluebutton.cms.gov/resources/variables/race",
                "code": "1",
                "display": "White"
            }
        },
        {
            "url": "http://hl7.org/fhir/us/core/ValueSet/omb-race-category",
            "extension": [
                {
                    "url": "ombCategory",
                    "valueCoding": {
                        "system": "http://terminology.hg/CodeSystem/v3-NullFlavor",
                        "code": "UNK",
                        "display": "Unknown"
                    }
                },
                { "url": "text", "valueString": "Unknown" }
            ]
        },
```

Reply:
The extension url for the `omb-race-category` is wrong.  Should be: http://hl7.org/fhir/us/core/StructureDefinition/us-core-race

http://hl7.org/fhir/us/core/STU3.1.1/StructureDefinition-us-core-race.html